package/MDAnalysis/analysis/align.py

```diff
@@ -212,14 +222,14 @@ def rotation_matrix(a, b, weights=None):
 
     Parameters
     ----------
-    a : array-like
+    a : array_like
```

Comment:
Can you check with other packages I'm pretty sure there should be a normal hyphen.